graylog2-server/src/test/java/org/graylog/plugins/views/search/export/SimpleMessageChunkTest.java

```diff
@@ -45,8 +48,8 @@ void valuesInOrderContainsMissingFieldsAsNull() {
                 linkedHashSetOf("timestamp", "source"),
                 msg1, msg2);
 
-        assertThat(sut.getAllValuesInOrder()).containsExactly(
-                new Object[]{"2015-01-01 01:00:00.000", "source-1"},
-                new Object[]{"2015-01-02 01:00:00.000", null});
+        assertThat(sut.getAllValuesInOrder())
+                .contains(new Object[]{"2015-01-01 01:00:00.000", "source-1"}, Index.atIndex(0))
+                .contains(new Object[]{"2015-01-02 01:00:00.000", null}, Index.atIndex(1));
```

Comment:
```suggestion
        assertThat(Arrays.stream(sut.getAllValuesInOrder()).toList()).asList()
                .containsExactly(
                        new Object[]{"2015-01-01 01:00:00.000", "source-1"},
                        new Object[]{"2015-01-02 01:00:00.000", null}
                );
```

Reply:
how about this?

Reply:
@mpfz0r What's the reason for the suggestion? Is it more straightforward for you in that way?

Reply:
a little, yeah. at least from the assertj point of view